### ISSUE TITLE
vim integration issues

### ISSUE THREAD

**Issue by tristanwietsma** [jimhester/lintr#25]
Syntastic is complaining:

``` vim
syntastic: error: checker r/lintr returned abnormal status 1
```

Any thoughts? I'm guessing this is a path issue...
- I've got lintr enabled in the vimrc per README:

``` vim
...
" Syntastic
let g:syntastic_r_checkers = 1
let g:syntastic_enable_r_lintr_checker = 1
let g:syntastic_r_lintr_linters = "with_defaults(line_length_linter(80))"
```
- I can load the lintr library in R
- I've got the lintr.vim in the syntax checkers directory:

``` bash
$ tree .vim/bundle/syntastic/syntax_checkers/r
.vim/bundle/syntastic/syntax_checkers/r
├── lint.vim
├── lintr.vim
└── svtools.vim
```
- `:echo executable('R')` gives 1

Here is the debug output from syntastic:

``` vim
"EngineXLSX.R" [dos] 659L, 26539C written                                                                                                                                                                                       
syntastic: 56.585804: &shell = '/bin/bash', &shellcmdflag = '-c', &shellpipe = '2>&1| tee', &shellquote = '', &shellredir = '>%s 2>&1', &shellslash = 0, &shelltemp = 1, &shellxquote = '', &shellxescape = ''                  
syntastic: 56.586569: UpdateErrors (auto): default checkers                                                                                                                                                                     
syntastic: 56.587531: CacheErrors: default checkers                                                                                                                                                                             
syntastic: 56.589386: g:syntastic_aggregate_errors = 0                                                                                                                                                                          
syntastic: 56.589759: getcwd() = /Users/tristan.wietsma/repos/ReportEngine                                                                                                                                                      
syntastic: 56.590313: CacheErrors: Invoking checker: r/lintr                                                                                                                                                                    
syntastic: 56.590954: SyntasticMake: called with options: {'errorformat': '%W%f:%l:%c: style: %m,%W%f:%l:%c: warning: %m,%E%f:%l:%c: error: %m,', 'makeprg': 'R --slave --restore --no-save -e ''library(lintr); lint(cache = TR
UE, commandArgs(TRUE), with_defaults(line_length_linter(80)))'' --args EngineXLSX.R', 'returns': [0]}                                                                                                                           
syntastic: 57.536184: checker output: ['Error in body(fun) : object ''fun'' not found', 'Calls: lint ... lapply -> FUN -> <Anonymous> -> findLocalsList -> body', 'Execution halted', '']                                       
syntastic: error: checker r/lintr returned abnormal status 1                                                                                                                                                                    
syntastic: 57.537370: r/lintr raw: []                                                                                                                                                                                           
syntastic: 57.537783: quiet_messages filter: {}                                                                                                                                                                                 
syntastic: 57.538355: aggregated: {'_sorted': 0, '_name': '', '_owner': 1, '_columns': 1, '_rawLoclist': []} 
```
